src/Umbraco.Web.UI.Client/src/views/datatypes/datatype.edit.controller.js

```diff
@@ -71,6 +60,7 @@ function DataTypeEditController($scope, $routeParams, appState, navigationServic
                 vm.preValuesLoaded = true;
                 vm.content = data;
 
+                console.log(vm.content.preValues);
```

Comment:
👀 @patrickdemooij9 Spotted a `console.log` left in.

Reply:
Whoops! I've removed it